client/my-sites/domains/domain-management/domain-connect/domain-connect-authorize.jsx

```diff
@@ -1,7 +1,8 @@
 /**
  * External dependencies
- */
-import React, { Component, PropTypes } from 'react';
+*/
```

Comment:
OMG WHY DID YOU REMOVE THIS SPACE?!?!?!?!111 What does it all _mean_? 
....
:trollface: 
Seriously, though, probably best to keep it consistently formatted across files.

Reply:
Huh. I definitely didn't mean to do that.  Will fix. Thanks!